src/gpl/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 
 find_package(Eigen3 REQUIRED)
 
+#enable_language(CUDA)
```

Comment:
is this meant to be commented out here, or is this some leftover debugging change ?

Reply:
This is the leftover change, I have deleted this line.